org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/A4Preferences.java

```diff
@@ -16,6 +30,11 @@
 
 import edu.mit.csail.sdg.translator.A4Options.SatSolver;
 
+/**
+ *
+ * @modified Nuno Macedo // [HASLab] electrum-decomposed
```

Comment:
simply adds the `DecomposedPref` preference for selecting the decomposed solving strategy